piccolo_admin/example.py

```diff
@@ -322,6 +322,10 @@ async def booking_endpoint(request, data):
         Movie.release_date,
     ],
     rich_text_columns=[Movie.description],
+    read_only_columns=[
+        Movie.release_date,
+        Movie.won_oscar,
+    ],
```

Comment:
One thing we might want to do is create a new table just for testing read only columns. I did something similar for nullable columns:

https://github.com/piccolo-orm/piccolo_admin/blob/2beacfe3d0285fcf0e09a1141ad26121906a91f5/piccolo_admin/example.py#L213-L222

So we have a table with every possible column type, so we can quickly see that they all work. It'll also make it easier when writing Playwright tests.

Reply:
That's a great idea. I will try to make that table with all possible column types and test it.

Reply:
Thanks